multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -459,6 +461,17 @@ def transform_label(
             y = self._label_scaler.transform(np.expand_dims(y, axis=-1))[:, 0].astype(np.float32)
         elif self.label_type == ROIS:
             y = y_df  # Do nothing. TODO: Shall we transform this?
+        elif self.label_type == NER_ANNOTATION:
+            text_column_index = 0  # Currently, we only support one text column.
+            x_df = df[self._text_feature_names[text_column_index]]
+            y = self._label_generator.transform(y_df)
+            x = self.transform_text(df)
```

Comment:
Calling `transform_text` inside `transform_label` looks kind of hacky since they are supposed to independent. How about adding a `transform_multimodal()` class function to handle the cases where we need to transform more than one modalities at the same time?

Reply:
I added a TODO here and I will do the refactor in the next PR this week.